lib/common/common.go

```diff
@@ -440,18 +440,10 @@ func writeACI(layer io.ReadSeeker, manifest schema.ImageManifest, curPwl []strin
 }
 
 func getExecCommand(entrypoint []string, cmd []string) appctypes.Exec {
-	var command []string
 	if entrypoint == nil && cmd == nil {
```

Comment:
Hm, I pushed an old version, this is not needed. If both `entrypoint` and `cmd` are nil, `append(entrypoint, cmd...)` returns `nil` already.